README.md

```diff
@@ -14,8 +14,9 @@ This project was tested against these Druid versions:
 
 This repository uses Maven and requires at least Java 11 to build:
 
-        mvn clean package
+        mvn --activate-profiles druid-31.0.1 clean package
```

Comment:
Does this really work?
The [documented way](https://maven.apache.org/guides/introduction/introduction-to-profiles.html#Explicit_profile_activation) of activating profiles is by using "-P"

Reply:
It's the same argument ;)
```
 -P,--activate-profiles <arg>            Comma-delimited list of profiles to activate
```
I +think* it's best practice to avoid argument shortcuts in scripts/tests, but I might be wrong. So no strong opinion

Reply:
I usually agree but I guarantee that almost no one will know the long-form in this case. The `-P` is pretty much standard everywhere.

Reply:
Changed in aa5ffe1